projects/1.18/assets/industrial-foregoing/industrialforegoing/patchouli_books/industrial_foregoing/zh_cn/entries/agr_hus/hydroponic_bed.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "水培床",
+  "icon": "industrialforegoing:hydroponic_bed",
+  "category": "industrialforegoing:agriculture_husbandry",
+  "sortnum": 16,
+  "read_by_default": true,
+  "pages": [
+    {
+      "type": "text",
+      "text": "这台机器在提供能量和$(6)水$()（对于下界作物来说是$(6)熔岩$()），它就能略微提升作物的生长速度。$(br2)通过提供非常少量的$(l:resource_production/ether_gas)以太气体$()，使作物的生长速度大幅提升，并且机器还会自动重新种植。"
```

好像前半句有点不通顺？
稍微改了一下，不一定合适
```suggestion
      "text": "这台机器在提供能量和$(6)水$()（对于下界作物来说是$(6)熔岩$()）时，可以略微提升作物的生长速度。$(br2)若提供微量的$(l:resource_production/ether_gas)以太气体$()，可使作物的生长速度大幅提升，且此时机器还会自动补种。"
```

---

嗯，改之后好很多。